### ISSUE TITLE
Add a search feature in the sample app

### ISSUE THREAD

**Issue by JoanZapata** [JoanZapata/android-iconify#103]

**Comment by MarcinOrlowski** [comment]
Actually it would be much more useful to have project website with all the icons and have search there. I'd bet more devs would praise such approach over search in demo app.

**Comment by Pedramrn** [comment]
+1
Start working with iconify app today, stunned by variety of icons!!! How am I supposed to find the backup icon?!! 
I'm not a front-end developer, but if the app was open sourced, I could make a PR for that.

**Comment by keunes** [comment]
+trillion for a search function in the demo app!

I personally prefer to check the app over going online when looking for an icon (for a mobile app project)

**Comment by Pedramrn** [comment]
I really would love to contribute in this feature. Unfortunately I don't have much time.
 I saw some `enum`s, where can I find the names and ids?
Any suggestion from the owner to make this quicker?

**Comment by MarcinOrlowski** [comment]
@Pedramrn what's the point of your comment really? If you want to contribute, fork, modify and send pull request. Now you just asked elementary questions.

**Comment by JoanZapata** [comment]
@MarcinOrlowski That's inappropriate.

@Pedramrn [Here](https://github.com/JoanZapata/android-iconify/blob/master/android-iconify-sample/src/main/java/com/joanzapata/iconify/sample/Font.java) is a good starting point. That's how the sample app lists all icons. Although I should warn you that I might do it myself soon.

**Comment by Pedramrn** [comment]
@MarcinOrlowski Thanks. I let you know If I start working on it.
:+1:

**Comment by baurine** [comment]
Hi guys, I just want to say, I really like this project and used in most of my  android projects, search feature is very useful for me, so I added it by myself, and have send the pull request #166 . If you guys think it is ok, I plan to do more, like, when you click a icon, it will display some similar icons.

**Comment by MarcinOrlowski** [comment]
No, it is not working correctly as activity is not resized when IME opens, therefore some icons remain invisible. Yet is shall be sufficient to edit Manifest to fix that
